WordPress/Classes/Extensions/PHAsset+Exporters.swift

```diff
@@ -0,0 +1,281 @@
+import Foundation
+import Photos
+import MobileCoreServices
+import AVFoundation
+
+extension PHAsset {
+    
+    typealias SuccessHandler = (resultingSize: CGSize) -> ()
+    typealias ErrorHandler = (error: NSError) -> ()
+
+    /**
+     Exports an asset to a file URL with the desired targetSize and removing geolocation if requested. 
+     The targetSize is the maximum resolution permited, the resultSize will normally be a lower value that maitains the aspect ratio of the asset
+     
+     - Parameters:
+        - url: file url to where the asset should be exported, this must be writable location
+        - targetUTI: the UTI format to use when exporting the asset
+        - maximumResolution:  the maximum pixel resolution that the asset can have after exporting. If CGSizeZero is provided the original size of image is returned.
+        - stripGeoLocation: if true any geographic location existent on the metadata of the asset will be stripped
+        - successHandler:  a handler that will be invoked on success with the resulting resolution of the asset exported
+        - errorHandler: a handler that will be invoked when some error occurs when generating the exported file for the asset
+     */
+    func exportToURL(url: NSURL,
+        targetUTI: String,
+        maximumResolution: CGSize,
+        stripGeoLocation: Bool,
+        successHandler: SuccessHandler,
+        errorHandler: ErrorHandler) {
+        
+        switch self.mediaType {
+        case .Image:
+            exportImageToURL(url,
+                targetUTI: targetUTI,
+                maximumResolution:maximumResolution,
```

Comment:
Code style - spaces after colon in all method parameters of the function calls in this switch